spacialgaze-plugins/SG.js

```diff
@@ -116,6 +116,22 @@ SG.reloadCSS = function () {
 	http.get(options);
 };
 
+//Daily Rewards System for SpacialGaze by Lord Haji
+SG.giveDailyReward = function (userid, user) {
+	if (!user || !userid) return false;
+	userid = toId(userid);
+	if (!user.autoconfirmed) return false;
+	if (!Db('DailyBonus').has(userid)) Db('DailyBonus').set(userid, 1);
+	let seen = Db('seen').get(userid);
+	if ((Date.now() - seen) < 86400000) return false;
+	if ((Date.now() - seen) > 89964000) Db('DailyBonus').set(userid, 1);
```

Comment:
After testing it seems that everytime the user connects the get a daily bonus, which is not what we want of course.